Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -80,8 +75,8 @@ export class JavaFilesExtractor {
 
         if (this.win) {
             if ('.tar' === fileEnding) { // a simple tar
```

Comment:
Is this still a separate case than `isTar` below?

Reply:
Yes, it is a separate case.  For windows you have to run a 7zip extraction twice in order to extract the compressed tar types. Simple tar files we can simply pass to the 7zip extraction method once.